release_scripts/signassert.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.ALL import *
```

Comment:
I tried doing it, but I end up with the same problem. I moved it to a parser.py file. Then ArmoryUtils.py has `from parser import CLI_OPTIONS, CLI_ARGS`. So PyBtcWallet imports from parser.py indirectly, through its ArmoryUtils import, and still causes an issue where the parser.py options/arguments are recognized over the ones in the signing script.